includes/functions.php

```diff
@@ -4119,3 +4119,22 @@ function dokan_get_withdraw_threshold( $user_id ) {
 
     return ( $threshold_day ) ? absint( $threshold_day ) : 0;
 }
+
+/**
+ * Dokan Withdraw Limit Value Validation
+ *
+ * @param mixed $option_value
+ * @param mixed $option_name
+ *
+ * @since DOKAN_LITE_SINCE
+ *
+ * @return mixed $option_value
+ */
+function dokan_withdraw_limit_value_validation( $option_value, $option_name ) {
```

Comment:
By considering the issue, we also need to fix the issue on the admin panel where the admin sets the withdrawal amount to a negative value. Currently, the admin can set the withdrawal amount in two places.

- Vendor setup wizard
- Dokan admin panel

Let me know if there's any confusion.